spec/services/fixed_region_info_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "spec_helper"
+
+describe FixedRegionInfo do
+  it "calculates average, number of contributors, top contributors and number of samples for sessions for given stearm" do
```

Comment:
I'd suggest to try to refactor this test a bit. It covers the production code but does not really explain how the input is related to the output. You can think of tests as documentation and ask yourself, does the code and description of this test explain why the output is
```
average: 1.5,
number_of_contributors: 2,
top_contributors: (["Test User1", "Test User2"]),
number_of_instruments: 2,
number_of_samples: 120,
```
?